src/toast/ops/scan_map/kernels.py

```diff
@@ -84,10 +84,10 @@ def scan_map(
 
 @kernel(impl=ImplementationType.COMPILED, name="scan_map")
 def scan_map_compiled(*args, use_accel=False):
-    return scan_map_compiled(*args, use_accel)
+    return libtoast_scan_map(*args, use_accel)
 
 
-def scan_map_compiled(
+def libtoast_scan_map(
```

Comment:
I would like to keep the name of this function as `scan_map_compiled`, because it is a function defined here in python, not an imported name from libtoast.

Reply:
I avoided `scan_map_compiled` to avoid having two function having the sane name, do you have a suggested other name?

Reply:
Ah, did not notice from the highlighted line that we had duplicate naming...  Ok, then leave it as it is.